src/models/ITestStation.ts

```diff
@@ -1,6 +1,7 @@
 export interface ITestStation {
     testStationId: string;
     testStationPNumber: string;
+    testStationStatus?: string | null;
```

Comment:
why would you need null with optional parameters 
type T1 = (x?: string) => string;              // x has type string | undefined

Reply:
We had a conversation on the same. Has been  removed